modules/examplepersist/examplepersist.go

```diff
@@ -63,12 +63,35 @@ var logChan chan string
 // returning and the module exiting.
 var handlerErrChan chan error
 
+// When the agent sends the persistent module it's configuration, it will come in via
+// the config channel as a JSON byte slice so we can unmarshal it into our configuration
+var configChan chan []byte
```

Comment:
I think passing the configuration through the call to `Run()` would be easier, or are you using a channel in case we want to change the configuration without reloading the module?